09. cli-and-http-request/clihelper/WriteUsingOutputLoggers.go

```diff
@@ -1,8 +1,10 @@
 package clihelper
 
-func WriteUsingOutputLoggers(c CliBindingProperties, d []byte) {
-	_, err := w.Writer(c.WritersCollection.WriterConfiguration.WriteToFilelocation, d)
+func WriteUsingOutputLoggers(cliBindingProperties CliBindingProperties, line string) {
+	_, err := w.Writer(line)
```

Comment:
Same goes here, where is `w` coming from?

Reply:
corected